pandas/tests/frame/test_arithmetic.py

```diff
@@ -374,14 +379,17 @@ def test_floordiv_axis0(self):
         result2 = df.floordiv(ser.values, axis=0)
         tm.assert_frame_equal(result2, expected)
 
-    @pytest.mark.slow
     @pytest.mark.parametrize("opname", ["floordiv", "pow"])
     def test_floordiv_axis0_numexpr_path(self, opname):
         # case that goes through numexpr and has to fall back to masked_arith_op
         op = getattr(operator, opname)
 
-        arr = np.arange(10 ** 6).reshape(100, -1)
```

Comment:
what does this change make a difference? why is this slow i the first place?

Reply:
pls don't resolve unless you answer the question. also typically let the question asker (me) resolve